test/fixtures/schema/object-fixed-type.apib

```diff
@@ -0,0 +1,10 @@
+# Resource [/example]
+
+## Action [GET]
+
++ Response 200 (application/json)
+
+    + Attributes
+        - person (object, fixed-type)
+            - first_name
+            - last_name
```

Comment:
Please add some values to the properties here and in https://github.com/apiaryio/drafter/blob/master/test/fixtures/schema/object-fixed.apib.